packages/clerk-js/src/ui/common/__tests__/authPropHelpers.test.ts

```diff
@@ -0,0 +1,15 @@
+import { normalizeRoutingOptions } from '../../../utils/authPropHelpers';
+
+describe('auth prop helpers', () => {
+  describe('normalizeRoutingOptions', () => {
+    it("returns routing: 'path' if path was provided and no routing was provided", () => {
+      expect(normalizeRoutingOptions({ path: 'test' })).toEqual({ routing: 'path', path: 'test' });
+    });
+
+    it('returns the same options if routing was provided (any routing) and path was provided (avoid breaking integrations)', () => {
```

Comment:
❓ i thought we wanted to introduce this breaking change and force everyone to use the correct options.
cc: @nikosdouvlis